### PR TITLE
FEC-10931 Add support for KalturaPlaybackCaption in OVP providers

### PR DESCRIPTION
**Pull request by chausov**
Solves FEC-10931, FEC-10727
Add support for KalturaPlaybackCaption in OVP providers

### CODE REVIEW

**Comment by chausov** [comment]
Add boolean useApiCaptions by default false.

File: Sources/OVP/Provider/OVPMediaProvider.swift [hunk 1]
```diff
@@ -444,6 +445,41 @@ public enum OVPMediaProviderError: PKError {
         return playURL
     }
     
+    private func createExternalSubtitles(ovpPlaybackContext context: OVPPlaybackContext?, ks: String?) -> [PKExternalSubtitle]? {
+        
+        if let playbackCaptions = context?.playbackCaptions {
+            
+            return playbackCaptions.compactMap({
+                var webVttUrl = $0.webVttUrl
+                // if let flavors = mediaSources.flavors, flavors.isEmpty {
```
Comment (x-NR-x) [inline Sources/OVP/Provider/OVPMediaProvider.swift]: If this is not needed please clean.

**Review by x-NR-x — APPROVED** [review]
Just cleanup unneeded commented code.

**Review by x-NR-x — CHANGES_REQUESTED** [review]
Missing _'useApiCaptions'_ to enable/disable the feature.

**Review by x-NR-x — APPROVED** [review]

**Pull request merged**